### PR TITLE
chore(github): update issue template to hide comments

### PR DESCRIPTION
**Pull request by zoechi**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Other... Please describe: GitHub issue template
```

**What is the current behavior?** (You can also link to an open issue here)

Created issues contain the additional explanation which results in a lot of noise in created issues

**What is the new behavior?**

Additional explanation will only be shown while an issue is edited.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

Wrap additional descriptions in comments so that they are hidden in created issues (only visible during editing)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -5,23 +5,21 @@
...
 
 **Reproduction of the problem**
-If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem via https://plnkr.co or similar (you can use this template as a starting point: http://plnkr.co/edit/tpl:AvJOMERrnz94ekVua0u5).
+<!-- If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem via https://plnkr.co or similar (you can use this template as a starting point: http://plnkr.co/edit/tpl:AvJOMERrnz94ekVua0u5). -->
 
 **What is the motivation / use case for changing the behavior?**
 
 **Please tell us about your environment:**
 
 * **Angular version:** 2.0.0-rc.X
```
Comment (vicb) [inline .github/ISSUE_TEMPLATE.md]: could you add comments to those bullet points - often people leave the text unmodified. Thanks

Comment (zoechi) [inline .github/ISSUE_TEMPLATE.md]: You think an additional comment to 

**What is the motivation ...**
**Please tell us ...*
**Angular version:**

would make it more likely for people to fill them?

**Comment by vicb** [comment]
LGTM please squash

**Comment by zoechi** [comment]
I created a new one #11473 so I don't need to check out the repo for squashing locally

**Pull request closed without merging**